### ISSUE TITLE
Gerbers don't have a NC drill file!

### ISSUE THREAD

**Issue by GeorgeRudolf** [Staacks/wifi-game-boy-cartridge#1]
No drill output has been generated in KiCad - using the Gerbers currently hosted on github will result in failure.

**Comment by Staacks** [comment]
Thanks, I have added those now.

Note, that I had generated the Gerber files without testing them as I ordered my PCBs from Aisler and they directly use the KiCad project file. I just placed an order with JLCPCB now using the Gerber files, so if you want a confirmation first, you can wait until I received the result.

Another, entirely unrelated note, but as you seem on the verge of building it now before I can give an update on this issue, you might find this detail interesting:
I am currently looking a bit into the power issues I mentioned at the end of the video and it seems to be two-fold: One issue is a voltage drop for the Game Boy when the ESP uses WiFi intermittently, which seems to be less of an issue when the WiFi usage is more or less constant (like when streaming video in my upcoming follow up). The other issue is that the voltage regulator I used has a rather high dropout voltage and fails to deliver the 3.3V if the batteries are not 100% full or if NiMh rechargables are being used. I am looking for a pin-compatible alternative right now, but unfortunately as is the case so often at the moment, most are not available.

**Comment by GeorgeRudolf** [comment]
What's your test platform? DMG, MGB, or CGB?
I can have a look at the power rails on DMG and on a GB Boy Colour, when I get it built.

Unfortunately, yes, sourcing regulators has become a nuisance.

**Comment by Staacks** [comment]
I only have a DMG. The problem has two sides:
- When I turn on the Game Boy, the Nintendo logo is often surrounded by glitches that form a repeating pattern along the tile grid. I think this is due to a voltage drop as the ESP starts while the Game Boys voltage supply is not entirely stabilized, corrupting some of its memory. I think that this would require delaying either the start of the ESP or the Game Boy (via Reset pin?), but this problem is mostly cosmetic as I do not see any problems later and the glitches disappear after the logo screen.
- If the battery voltage is a bit too low (and I am talking about something like 1.48 V per battery), the ESP does not boot reliably. If it is even lower (especially with NiMh batteries), I cannot get the ESP to boot at all. I have not yet measured the actual voltage at the cartridge at this point (yeah, it's a matter of seconds, but I do not have the Game Boy at hand right now), but I think that I underestimated how much the 5V from the Game Boy actually depend on the source voltage and that the dropout voltage of the NCP1117 is just too high to sustain a proper voltage for the ESP in that case. After a quick search I now ordered a few AP7361C-33ER as a pin-compatible replacement with lower drop-out voltage.
-
